mixer/pkg/runtime/config/config_test.go

```diff
@@ -529,6 +529,191 @@ Attributes:
 `,
 	},
 
+	{
+		Name: "basic handler cr config",
+		Events1: []*store.Event{
+			{
+				Key: store.Key{
+					Name:      "a1",
+					Namespace: "ns",
+					Kind:      "handler",
+				},
+				Type: store.Update,
+				Value: &store.Resource{
+					Spec: &configpb.Handler{
+						Name:            "a1",
+						CompiledAdapter: "adapter1",
+					},
+				},
+			},
+		},
+		E: `
+ID: 0
+TemplatesStatic:
+  Name: apa
+  Name: check
+  Name: quota
+  Name: report
+AdaptersStatic:
+  Name: adapter1
+  Name: adapter2
+HandlersStatic:
+  Name:    a1.ns
+  Adapter: adapter1
+  Params:  &Struct{Fields:map[string]*Value{},XXX_unrecognized:[],}
+InstancesStatic:
+Rules:
+Attributes:
+  template.attr: BOOL
+`,
+	},
+	{
+		Name: "basic handler cr config with params",
+		Events1: []*store.Event{
+			{
+				Key: store.Key{
+					Name:      "a2",
+					Namespace: "ns",
+					Kind:      "handler",
+				},
+				Type: store.Update,
+				Value: &store.Resource{
+					Spec: &configpb.Handler{
+						Name:            "a2",
+						CompiledAdapter: "adapter2",
+						Params:          adapter2Params,
+					},
+				},
+			},
+		},
+		E: `
+ID: 0
```

Comment:
Please let's not encourage this text-based validation. It's difficult to track references and _feels_ fragile. Why don't we use structural struct comparison?